tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -431,6 +431,105 @@ def test_get_metric_extraction_config_with_apdex(default_project):
         }
 
 
+@django_db_all
+@pytest.mark.parametrize("quality", ["good", "meh", "poor", "any"])
+@pytest.mark.parametrize("measurement", ["measurements.lcp"])
+def test_get_metric_extraction_config_with_count_web_vitals(default_project, quality, measurement):
+    with Feature({ON_DEMAND_METRICS: True, ON_DEMAND_METRICS_WIDGETS: True}):
+        create_widget(
+            [f"count_web_vitals({measurement}, {quality})"],
+            "transaction.duration:>=1000",
+            default_project,
+        )
+
+        config = get_metric_extraction_config(default_project)
+
+        assert config
+        assert len(config["metrics"]) == 1
+
+        if quality == "good":
+            assert config["metrics"][0] == {
+                "category": "transaction",
+                "condition": {"name": "event.duration", "op": "gte", "value": 1000.0},
+                "field": None,
+                "mri": "c:transactions/on_demand@none",
+                "tags": [
+                    {
+                        "condition": {
+                            "name": f"event.{measurement}.value",
+                            "op": "gte",
+                            "value": 2500,
```

Comment:
If you could import `VITAL_THRESHOLDS` and use the associated values it would make it easier for the reader why this specific number.